CondFormats/HGCalObjects/src/T_EventSetup_HeterogeneousHGCalHEFCellPositionsConditions.cc

```diff
@@ -0,0 +1,3 @@
+#include "RecoLocalCalo/HGCalRecProducers/plugins/HeterogeneousHGCalHEFCellPositionsConditions.h"
```

Comment:
If you look at the files marked with "invalid state" above (created and then deleted, polluting the commit history), two of them are the following:

- ```CondFormats/HGCalObjects/interface/HeterogeneousHGCalHEFCellPositionsConditions.h```

- ```CondFormats/HGCalObjects/src/HeterogeneousHGCalHEFCellPositionsConditions.cc```

This solution did not work, since, as discussed multiple times with @felicepantaleo, the code did not compile when mixing more than one subpackage. That is why, for the moment, much of the code is stored under ```RecoLocalCalo/HGCalRecProducers```.
I will address this point as soon as most of the others are taken care of.